src/FileSystem.h

```diff
@@ -80,4 +80,8 @@ class FileSystem : public ObjectCounter<FileSystem>
 
 		/// \todo ideally, this method would never be needed by client code!!
 		std::string getUserDir();
+
+		/// \brief Get the user-and-app-specific path where files can be written.
+		/// \todo ideally, this method would never be needed by client code!!
```

Comment:
does this TODO make sense? I'm not really sure what the aim is, here

Reply:
It is a copy&paste todo from method above. I'm not really sure, too. We should drop it.